Homework_2/task_2.py

```diff
@@ -0,0 +1,19 @@
+# Напишите программу, которая принимает две строки вида “a/b” - дробь с числителем и знаменателем. Программа должна возвращать сумму и произведение* дробей.
+# Для проверки своего кода используйте модуль fractions.
+
+import fractions
+
+f_frac = (str(input("Введите первую дробь в формате a/b ")))
+s_frac = (str(input("Введите вторую дробь в формате a/b ")))
```

Comment:
Функция `input` уже возвращает стоковый тип данных.
```suggestion
f_frac = input("Введите первую дробь в формате a/b ")
s_frac = input("Введите вторую дробь в формате a/b ")
```